vignettes/howto-dm-df.Rmd

```diff
@@ -147,15 +149,24 @@ We can use `dm_draw()` at any stage of the process to generate a visual represen
 ```{r}
 flights_dm_no_keys %>%
   dm_draw(rankdir = "TB", view_type = "all")
+```
+
 
+```{r}
```

Comment:
I wonder what the advantage of splitting into multiple chunks is.

Reply:
IMO, these empty spaces look a bit awkward. 

<img width="813" alt="Screenshot 2022-06-15 at 09 05 20" src="https://user-images.githubusercontent.com/11330453/173764229-d1b7eefb-aa1a-4483-948f-520e2b014cb6.png">

With multiple chunks, it looks much cleaner.

But happy to revert if you wish.